public/SBAdmin/js/sb-admin-2.js

```diff
@@ -6,7 +6,7 @@
     $("body").toggleClass("sidebar-toggled");
     $(".sidebar").toggleClass("toggled");
     if ($(".sidebar").hasClass("toggled")) {
-      $('.sidebar .collapse').collapse('hide');
+      $(".sidebar .collapse").collapse('hide');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ahmadsakur/kokeru/pullRequest?prid=8259641)
```suggestion
      $(".sidebar .collapse").collapse("hide");
```